pkg/apiserver/apiserver.go

```diff
@@ -117,14 +122,38 @@ func runE(flags *flags, log logr.Logger) error {
 	if err != nil {
 		return fmt.Errorf("creating client: %w", err)
 	}
+	dynamicClient, err := dynamic.NewForConfig(cfg)
+	if err != nil {
+		return fmt.Errorf("creating dynamic client: %w", err)
+	}
+
+	ctx := context.Background()
+	certPool := x509.NewCertPool()
+	certs, err := ioutil.ReadFile(flags.TLSCertFile)
+	if err != nil {
+		return err
+	}
+	certPool.AppendCertsFromPEM(certs)
+	grpcTLSConfig := &tls.Config{
+		InsecureSkipVerify: true,
+		RootCAs:            certPool,
+	}
+	// Create grpc client for watch api
+	grpcClient, err := grpc.DialContext(ctx, flags.address, grpc.WithTransportCredentials(credentials.NewTLS(grpcTLSConfig)))
+	if err != nil {
+		return err
+	}
```

Comment:
extract this into `createInternalGRPCClient` method.

Reply:
done